angular-workspace/projects/ni/nimble-angular/src/thirdparty/directives/router_link.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * [Nimble]
```

Comment:
The most substantive diffs seem to be in router code. Is there any focused testing we should do to verify it's not breaking anything? I'm not very familiar with our auto test coverage in this area but I know we've had many rounds of trying to get router behaviors just right so these changes make me slightly nervous.

Reply:
Really the only effective change to `router_link.ts` is the way input values are coerced to boolean for `preserveFragment`, `skipLocationChange`, and `replaceUrl`. The latter is pretty universally used in SLE anchor tabs to prevent excess navigation history from accumulating. I went ahead and verified that in LabManagement, switching between tabs does not add to the history. I think that's probably sufficient.